cmake/configs/default.cmake

```diff
@@ -35,6 +35,7 @@ hunter_config(Android-SDK-Platform VERSION 21_r02) # API 21
 hunter_config(Android-SDK-Platform-tools VERSION r22)
 hunter_config(Android-SDK-Tools VERSION 24.3.3)
 hunter_config(Android-Support-Repository VERSION 16)
+hunter_config(Avahi VERSION 0.6.31 CMAKE_ARGS BUILD_SHARED_LIBS=ON)
```

Comment:
You can use `hunter_cmake_args` in `hunter.cmake` module of the Avahi package so we can keep config file clean. See [wiki](https://github.com/ruslo/hunter/wiki/usr.adding.new.package#cmake-options) and [example](https://github.com/ruslo/hunter/blob/master/cmake/projects/Comet/hunter.cmake).